.circleci/config.yml

```diff
@@ -0,0 +1,253 @@
+version: 2.1
+executors:
+  default:
+    docker:
+    - image: circleci/golang:1.15.8
+
+aliases:
+- &restore_cache
+  restore_cache:
+    key: go-mod-v1-{{ checksum "go.sum" }}
+- &save_cache
+  save_cache:
+    key: go-mod-v1-{{ checksum "go.sum" }}
+    paths:
+      - "/go/pkg/mod"
+
+jobs:
+  check-style:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - *restore_cache
+    - run: make check-style
+    - *save_cache
+
+  check-modules:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - *restore_cache
+    - run: make check-modules
+    - *save_cache
+
+  check-boilerplate:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - run: ./scripts/verify-boilerplate.sh
+
+  check-mocks:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - run: make verify-mocks
+
+  test-sqlite:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - *restore_cache
+    - run: |
+        make unittest
+        make goverall
+    - *save_cache
+
+  test-postgres:
+    docker:
+    - image: circleci/golang:1.14.6
```

Comment:
We use on purpose older version of golang?

Reply:
Nope mistake. Fixed ;)